parm/parm_fv3diag/diag_table.ocn

```diff
@@ -0,0 +1,71 @@
+"ocn%4yr%2mo%2dy%2hr",    6,  "hours",  1,  "hours",  "time",  6,  "hours",  "1901 1 1 0 0 0"
```

Comment:
When combining the two atm & ocn diag file segments, do these two lines need to be at the top of the file along with the 
```
"fv3_history",    0,  "hours",  1,  "hours",  "time"
"fv3_history2d",  0,  "hours",  1,  "hours",  "time"
```
section?

Reply:
No. 

See the File Description and Field Description slides from https://data1.gfdl.noaa.gov/summer-school/Lectures/July16/03_Seth1_DiagManager.pdf
The only requirement is that the file be described before using it for a field.

Reply:
Thanks!